persist/src/main/resources/db/migration/V1__Initial_Import.sql

```diff
@@ -1,3 +1,12 @@
+CREATE TABLE IF NOT EXISTS countries
```

Comment:
What data do you think we'll want to store on the country-level? We might be able to just use an enum if there's no country-specific data to store.

Reply:
I forget that we can confine data to enums with jooq